### ISSUE TITLE
Restrict access to conference controller actions

### ISSUE THREAD

**Issue by klappradla** [rails-girls-summer-of-code/rgsoc-teams#840]
As mentioned in #829, restrict the access to the `conferences#new` (and related `create` etc.) actions to those users who have the ability to do this. Use `cancancan` to ensure this on the controller level.

**Comment by klappradla** [comment]
Hey @juuh42dias & @branquinhoaa, I think we totally forgot this one _(it was a follow up for PR #829 )_ - can you please check if this is still valid and then take care of it? It should only be a tiny task :v:

**Comment by branquinhoaa** [comment]
Ok @klappradla. We will take care of this.

**Comment by juuh42dias** [comment]
@klappradla current ability:

`can :crud, Conference if user.admin? || user.current_student?`

That's right?

**Comment by klappradla** [comment]
Yep, as far as I remember you changed it to this yourself @juuh42dias ✌️

**Comment by juuh42dias** [comment]
@klappradla 
"...changed it to this yourself"
'-' Sorry, I don't understand

**Comment by klappradla** [comment]
😹  no worries. Just wanted to confirm: yes, that's the current ability and it should be right.

**Comment by juuh42dias** [comment]
:+1:
